src/content/docs/fr/core-concepts/sharing-state.mdx

```diff
@@ -134,7 +136,7 @@ import AddToCartForm from '../components/AddToCartForm';
 <body>
   <header>
     <nav>
-      <a href="/">Vitrine Astro</a>
+      <a href="/">Astro storefront</a>
```

Comment:
I prefer the previous translation. You can keep this English version, but `Vitrine` do the job!
```suggestion
      <a href="/">Vitrine Astro</a>
```